index.html

```diff
@@ -394,10 +394,10 @@ <h2><dfn>observe()</dfn> method</h2>
         global object</a>.
         </li>
         <li>If <var>options</var>'s <a>entryTypes</a> and <a>type</a> members
-        are both omitted, then [=exception/throw=] a {{"SyntaxError"}}.
+        are both omitted, then [=exception/throw=] a {{"TypeError"}}.
         </li>
         <li>If <var>options</var>'s <a>entryTypes</a> is present and any other
```

Comment:
I noticed that currently Chrome does not throw an error in this case (so for example `observe({entryTypes: ['mark', 'measure'], buffered: true})` will not throw an error, we only issue a console warning. Thoughts?

Reply:
Sounds like Chrome should align its implementation. Might be good to add a counter that would help estimate breakage here